apps/studio/components/interfaces/Auth/EmailTemplates/EmailTemplates.tsx

```diff
@@ -43,32 +55,59 @@ export const EmailTemplates = () => {
               <EmailRateLimitsAlert />
             </div>
           ) : null}
-          <Card>
-            <Tabs_Shadcn_ defaultValue={TEMPLATES_SCHEMAS[0].title.trim().replace(/\s+/g, '-')}>
-              <TabsList_Shadcn_ className="pt-2 px-6 gap-5 mb-0 overflow-x-scroll no-scrollbar mb-4">
-                {TEMPLATES_SCHEMAS.map((template) => {
-                  return (
-                    <TabsTrigger_Shadcn_
-                      key={`${template.id}`}
-                      value={template.title.trim().replace(/\s+/g, '-')}
-                    >
-                      {template.title}
-                    </TabsTrigger_Shadcn_>
-                  )
-                })}
-              </TabsList_Shadcn_>
+          {isSecurityNotificationsEnabled ? (
+            <Card>
               {TEMPLATES_SCHEMAS.map((template) => {
-                const panelId = template.title.trim().replace(/\s+/g, '-')
+                const templateSlug = template.title.trim().replace(/\s+/g, '-').toLowerCase()
                 return (
-                  <TabsContent_Shadcn_ key={panelId} value={panelId} className="mt-0">
-                    <TemplateEditor key={template.title} template={template} />
-                  </TabsContent_Shadcn_>
+                  <CardContent key={`${template.id}`} className="p-0">
+                    <Link
+                      href={`/project/${projectRef}/auth/templates/${templateSlug}`}
+                      className="flex items-center justify-between hover:bg-surface-200 transition-colors py-4 px-6 w-full h-full"
+                    >
+                      <div className="flex flex-col">
+                        <h3 className="text-sm text-foreground">{template.title}</h3>
+                        {template.purpose && (
+                          <p className="text-sm text-foreground-lighter">{template.purpose}</p>
+                        )}
+                      </div>
+                      <ChevronRight
+                        size={16}
+                        className="text-foreground-muted group-hover:text-foreground transition-colors"
```

Comment:
I couldn’t get `group-hover:text-foreground transition-colors` to work. Feel free to remove.